components/cosmetic_filters/resources/data/element_picker.ts

```diff
@@ -463,6 +614,9 @@ const launchElementPicker = (root: ShadowRoot) => {
   svg.addEventListener(
     'mousemove',
     (event) => {
+      if (isAndroid) {
+        return
+      }
```

Comment:
if it's as easy as removing early returns, I'd leave this kind of mouse-specific logic in on Android since some folks would e.g. use a keyboard/mouse combo on a tablet

Perhaps it's worth using `pointer`/`hover` media queries rather than UA inspection? [ref](https://stackoverflow.com/a/52854585)